src/Console/BaseCommand.php

```diff
@@ -45,7 +45,7 @@ abstract class BaseCommand implements CommandInterface
      *
      * @var string
      */
-    protected $description = '';
+    protected static $description = '';
```

Comment:
why we need this property ?

Reply:
This sets the default value for the description so that all currently present Commands will return `''` when someone calls `MyCommand::getDescription()`

The static is needed so I don't have to instanciate an object via reflection (see above comment) inside the HelpCommand.

Reply:
Do you need the property at all? Userland code could implement the method, and the default could be `return '';`

Reply:
just like the `defaultName()` that you can have your own `defaultName()` implementation
we can have by default:
```php
    public static function getDescription(): string
    {
        return '';
    }
```
static property here is redundant IMO

Reply:
I personally liked the idea of overwriting the description property better for userland code instead of having to overwrite the whole `getDescription()` method but OK ^^

Reply:
@LordSimal property is useful only when it be public and we could document it